hoomd/hpmc/UpdaterMuVT.h

```diff
@@ -1979,11 +1971,8 @@ bool UpdaterMuVT<Shape>::moveDepletantsIntoNewPosition(unsigned int timestep, un
     #endif
 
     // initialize another rng
-    #ifdef ENABLE_MPI
-    hoomd::RandomGenerator rng(hoomd::RNGIdentifier::UpdaterMuVTDepletants2, timestep, this->m_seed, this->m_exec_conf->getPartition());
-    #else
-    hoomd::RandomGenerator rng(hoomd::RNGIdentifier::UpdaterMuVTDepletants2, timestep, this->m_seed);
-    #endif
+    hoomd::RandomGenerator rng(hoomd::Seed(hoomd::RNGIdentifier::UpdaterMuVTDepletants2, timestep, this->m_sysdef->getSeed()),
+                               hoomd::Counter(this->m_exec_conf->getPartition()));
```

Comment:
I noticed you used a ternary operator with `getPartition` some where else for non-MPI runs, but here you just do the call.

Reply:
Jens wrote the ternary operators. I tried to convert as much of the seeding code as-is without rewriting to avoid errors.